extensions/amp-story/1.0/development-ui.js

```diff
@@ -181,35 +193,29 @@ export class DevelopmentModeLog {
    * @return {?Element}
    */
   build() {
-    this.contextStringEl_ = this.win_.document.createElement('span');
-    this.contextStringEl_.classList.add(
-      'i-amphtml-story-developer-log-context'
+    this.contextStringEl_ = (
+      <span class="i-amphtml-story-developer-log-context"></span>
     );
-    const titleEl = this.win_.document.createElement('div');
-    titleEl.textContent = 'Developer logs for page ';
-    titleEl.appendChild(this.contextStringEl_);
 
     const closeDeveloperLogEl = createButton(
       this.win_,
       'i-amphtml-story-developer-log-close',
       () => this.hide()
     );
 
-    const headerEl = this.win_.document.createElement('div');
-    headerEl.classList.add('i-amphtml-story-developer-log-header');
-    headerEl.appendChild(titleEl);
-    headerEl.appendChild(closeDeveloperLogEl);
-
-    this.entriesEl_ = this.win_.document.createElement('ul');
-    this.entriesEl_.classList.add('i-amphtml-story-developer-log-entries');
+    this.entriesEl_ = <ul class="i-amphtml-story-developer-log-entries"></ul>;
 
-    this.root_ = this.win_.document.createElement('div');
-    this.root_.classList.add('i-amphtml-story-developer-log');
-    toggle(this.root_, false);
-    this.root_.appendChild(headerEl);
-    this.root_.appendChild(this.entriesEl_);
+    this.root_ = (
+      <div class="i-amphtml-story-developer-log" hidden>
+        <div class="i-amphtml-story-developer-log-header">
+          <div>Developer logs for page {this.contextStringEl_}</div>
+          {closeDeveloperLogEl}
+        </div>
+        {this.entriesEl_}
+      </div>
+    );
 
-    this.clear();
+    this.clear(); // ???
```

Comment:
I think we might be able to remove this file overall but I'll defer to @gmajoulet who knows if this is being used.

Reply:
+1 let's keep it and delete the file later